schema/defs.json

```diff
@@ -60,6 +60,9 @@
         "GID": {
             "$ref": "#/definitions/uint32"
         },
+		"Umask": {
```

Comment:
The indent here is a bit strange.  You can automatically format these files by running `make fmt` in the `schema` directory.